cli/cmd/build/config/config.go

```diff
@@ -15,13 +16,49 @@ type Artifact struct {
 	URL  string `yaml:"url"`
 }
 
+type Extension struct {
+	Name    string            `yaml:"name"`
+	Version string            `yaml:"version"`
+	Specs   map[string]string `yaml:"specs"`
+}
+
 type Config struct {
```

Comment:
createAt can be specified here but it doesnt have to be documented in the example build config